common-topics/api-keys.md

```diff
@@ -46,20 +46,16 @@ You will, therefore, need to use a different set of API keys for the test enviro
 
 💥 **Use caution with storing and transferring API keys** 💥
 
-* Never send API keys unencrypted by email.
-* If you accidentally share your API keys, you must generate new ones by clicking the *regenerate* button.
-  Update your integrations so they will not stop working.
+* Vipps MobilePay will never ask for your API keys, and you must keep them secret.
+* Do *not* send API keys unencrypted by email.
```

Comment:
```suggestion
* *Always* send API keys by encrypted email.
```

Reply:
Your revision is much better. I am recalling something I learned from the tone of voice session which was to always use the positive version of the sentence because people tend to remember the sentence without the "not".